Makefile

```diff
@@ -124,6 +129,22 @@ registry-login:
 build-image:
 	ko publish --base-import-paths --tags="${IMAGE_TAG}" ./cmd/$(IMGCTRL) 
 
+# downloads and unpacks helm binary in output directory
+install-helm:
+	mkdir -p $(OUTPUT_DIR)/helm-v$(HELM_VERSION)-$(SYS_OS)-$(SYS_ARCH)
+	curl -o $(OUTPUT_DIR)/helm.tar.gz \
+		https://get.helm.sh/helm-v$(HELM_VERSION)-$(SYS_OS)-$(SYS_ARCH).tar.gz
+	tar -xf $(OUTPUT_DIR)/helm.tar.gz -C $(OUTPUT_DIR)/helm-v$(HELM_VERSION)-$(SYS_OS)-$(SYS_ARCH)
+	rm $(OUTPUT_DIR)/helm.tar.gz
+
+# creates a yaml template of image-controller
+template:
+	mkdir -p $(OUTPUT_DIR)
+	helm template \
```

Comment:
This is running the helm you have installed in your system and not the one installed by `install-helm`.